pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -126,14 +126,17 @@ class ClassTypeHintMatch:
     "mock_device_tracker_conf": "list[Device]",
     "mock_get_source_ip": "None",
     "mock_zeroconf": "None",
+    "mock_yaml_config": "pathlib.Path",
```

Comment:
Because of how the pylint plugin works, I think here it os better to let users choose whether or not to use the namespace:
```suggestion
    "mock_yaml_config": "Path",
```